test/fixtures/os-linux-dep-graph.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
think it's better to  also have the word `scratch` in the fixture name

Reply:
👍 will change